pkgs/top-level/all-packages.nix

```
@@ -18021,6 +18021,8 @@ in
 
   quicksynergy = callPackage ../applications/misc/quicksynergy { };
 
+  quill = callPackage ../tools/security/quill { };
+
```

For supporting the build on darwin:
```suggestion
  quill = callPackage ../tools/security/quill { 
    inherit (darwin.apple_sdk.frameworks) Security;  
  };

```